plugins/inventory/lxd.py

```diff
@@ -0,0 +1,1107 @@
+            - The client certificate file path.
+            required: false
+            aliases: [ cert_file ]
+            default: $HOME/.config/lxc/client.crt
+            type: path
+        trust_password:
+            description:
+            - The client trusted password.
+            - You need to set this password on the lxd server before
+                running this module using the following command.
+                lxc config set core.trust_password <some random password>
+                See (https://www.stgraber.org/2016/04/18/lxd-api-direct-interaction/)
+            - If trust_password is set, this module send a request for
+                authentication before sending any requests.
+            required: false
+            type: str
+        state:
+            description: Filter the container according to the current status
+            required: false
+            type: str
+            default: none
+            choices: ['STOPPED', 'STARTING', 'RUNNING', 'none']
+        prefered_container_network_interface:
+            description:
+            - if a container has multiple network interfaces, which one is the prefered as pattern
+            - combined with the first number that can be found e.g. eth + 0
+            required: false
+            type: str
+            default: eth
+        prefered_container_network_family:
+            description:
+            - if a container has multiple network interfaces, which one is the prefered by family
+                - inet is ipv4
+                - inet6 is ipv6
+            required: false
+            type: str
+            default: inet
+            choices: ['inet', 'inet6']
+        groups:
+            description:
+            - Group the results
+                - 'location' grabed sample by web, maybe it works
+                - 'pattern' samba* or regex
+                - 'network_range' 192.168.0.0/24
+                - 'os' ubuntu
+                - 'release' groovy
+                - 'profile' default
+                - 'vlanid' works on defined networks e.g. br0 with vlanid_default == 666
+            required: false
+            type: json
+            default: none
+        selftest:
+            description:
+            - Load default data to test plugIn
+                - 'path' to file with test data e.g. ansible_collections/community/general/tests/integration/targets/inventory_lxd/files/lxd_inventory.atd
+                - 'activate' True
+            required: false
+            type: json
+            default: false
+        dumpdata:
+            description: dump out data to debug
+            required: false
+            type: bool
+            default: false
+'''
+
+EXAMPLES = '''
+# simple lxd.yml
+plugin: community.general.lxd
+url: unix:/var/snap/lxd/common/lxd/unix.socket
+selftest: False
+dumpdata: False
+
+# simple lxd.yml including filter
+plugin: lxd
+url: unix:/var/snap/lxd/common/lxd/unix.socket
+state: RUNNING
+selftest: False
```

Comment:
```suggestion
selftest: false
```
Booleans in YAML in docs should be `true` and `false`, not other variants.

Reply:
done